estudiantes1718.md

```diff
@@ -7,7 +7,7 @@
 <!-- aquí el listado de miembros del grupo 1 -->
 
 
-## Grupo 2. Igualdad de género
+## Grupo 2. Igualdad de género (https://github.com/orgs/DeustoPWEB/teams/grupo-2/members)
```

Comment:
Kaixo @mariarobledo96 , aquí, en cambio, sí tendrías que añadir los corchetes, y el enlace debería llevar a vuestro repositorio del grupo, en lugar de al listado de miembros. Tendría que ser así:

```
## [Grupo 2. Igualdad de género](https://github.com/DeustoPWEB/pweb-grupo-2)
```

De esa manera, el texto `Grupo 2. Igualdad de género` se convertirá en un enlace que llevará a la URL indicada. 

¡Podéis trabajar juntos @DeustoPWEB/grupo-2 en esta correción! Cuando lo tengáis, haced un commit y este pull request debería actualizarse automáticamente.